ParisBNF/abb/BNFabb105.xml

```diff
@@ -0,0 +1,979 @@
+                             <textLang mainLang="gez"/>
+                             <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ። አ</hi>
+                             ሐዱ᎓ አምላክ፨ ወእምዝ፡ ንጽሕፍ፡ ው<hi rend="rubric">ዳሴሃ᎓ ለእግዚእትነ᎓ ማርያም᎓ ድንግል፡ ወላ</hi>ዲተ᎓ አምላክ᎓ <gap reason="ellipsis" resp="DN"/>
+                             ተሰመይኪ᎓ ፍቅርተ፡ ኦቡርክት፡ እምአንስት፡ አንቲ፡ ይእቲ፡ ዳግሚት፡ ቀመር፡ እንተ፡ ተሰመይኪ፡ ቅድስተ፡ ቅዱሳን። ዘውስቴታ፡ ጽላተ፡ ኪዳን፡</incipit>
+                             <explicit xml:lang="gez">
+                             ኦእግዚእትነ᎓ እሙ᎓ ለዐቢይ፡ ወክቡር፡ አሐዜ፡ ኵሉ፡ ዓለም፡ ዘምሉእ፡ ሰማያ፡ ተ፡ ወምድር፡ ስብሐቲሁ፡ ሎቱ፡ ንገኒ፡ ወሎቱ፡ ነአምን፡
+                             እምይእዜ፡ ወእስከ፡ ለዓለም፡ ዓለም፡ አሜን፨ <hi rend="rubric">
+                             ሠረቀ᎓ በሥጋ᎓ እምድንግል᎓ ዘእን</hi></explicit>
+                           </msItem>
+
+                           <msItem xml:id="ms_i1.6">
+                             <locus from="181v" to="186ra"/>
+                             <title type="complete" ref="LIT4711Mastabqwe"/>
+                             <textLang mainLang="gez"/>
+                               <incipit xml:lang="gez"><hi rend="rubric">ውዳሴ᎓ ወግናይ᎓ ዘእግዚእትነ᎓ ማርያም᎓᎓</hi> ወላዲተ᎓
+                             አምላክ፨ ጸሎታ᎓ ወበረከታ᎓ የሀሉ᎓ ምስለ᎓ ገብራ᎓ <gap reason="lost"/>ለዓለመ᎓ ዓለም᎓ አሜን፨
+                             <hi rend="rubric">ቅድስት፡ ወብፅዕ</hi>ት᎓ ስብሕት᎓ ወቡርክት፨ ክብርት᎓ ወልዕልት፨ አንቀጸ᎓ ብርሃን᎓
+                             መዐርገ᎓ ሕይወት᎓</incipit>
+
+                             <msItem xml:id="ms_i1.6.1">
+                               <locus from="181v" to="18vb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ዱያን፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ኀዋጺሃ፡ ለነፍስ፡ ወመፈውሰ፡ ሥጋ፡
+                               </incipit>
+                             </msItem>
+
+                             <msItem xml:id="ms_i1.6.2">
+                               <locus from="181vb" to="182rb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ እለ፡ ይነግድ፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ <pb n="182ra"/>ኢየሱስ፡ ክርስቶስ፡ ዘይሁብ፡
+                                    ፍኖተ፡ ባሕር፡ ዘይሁብ፡ ፍኖተ፡ ባሕር፡ ወስተ፡ ባሕር፡
+                               </incipit>
+                             </msItem>
+
+                             <msItem xml:id="ms_i1.6.3">
+                               <locus from="182rb" to="182vb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ዝናማት፡</hi></title>
```

Comment:
ref="LIT4711Mastabqwe#SupplicationRain"? also the following?

Reply:
yes